src/coreclr/nativeaot/Runtime/eventtrace_etw.h

```diff
@@ -10,7 +10,39 @@
 
 #ifdef FEATURE_ETW
 
-#include "EtwEvents.h"
+#ifndef  HOST_UNIX
```

Comment:
I think we can just rely on this being under `FEATURE_ETW` without also checking `HOST_UNIX`.